microcosm_flask/conventions/relation.py

```diff
@@ -41,11 +42,56 @@ def configure_createfor(self, ns, definition):
         @response(definition.response_schema)
         def create(**path_data):
             request_data = load_request_data(definition.request_schema)
-            response_data = definition.func(**merge_data(path_data, request_data))
+            response_data = require_response_data(definition.func(**merge_data(path_data, request_data)))
```

Comment:
That surprised me. I'd have only expected this to be present for DELETE where the return value of the func would be expected to indicate if the resource couldn't be found on delete.